bitnami/airflow/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: airflow
-version: 6.5.1
+version: 6.5.2
```

Comment:
I think this would be a major change or at least a minor one, in fact, you are changing the structure of the values so users that want to update must perform extra actions.

If we want to be a minor change we will need to add retro compatibility to the old parameters. I can think about the following 

```
{{- if .Values.airflow.cloneDagFilesFromGit.repositories -}}
  git clone {{ .repository }} --branch {{ .branch }} /dags-clone-old-versions
{{- end -}}
```

Reply:
do you agree @marcosbc ?

Reply:
@dani8art Agree, I was also thinking the same. Will make the change. My change breaks backward compatibilty, we can have back ward compatibility keeping, `repository`, `branch` and `path` in `cloneDagFilesFromGit` intact and adding `repositories` there. Waiting for @marcosbc 's comment.

Reply:
@dani8art Agree, I was also thinking the same. Will make the change. My change breaks backward compatibilty, we can have back ward compatibility keeping, `repository`, `branch` and `path` in `cloneDagFilesFromGit` intact and adding `repositories` there. Waiting for @marcosbc 's comment.

Reply:
Yes, I think it is not needed to have it in `values.yaml` anymore because we can use the new `repositories` parameter, however, we should add defense `if` in order to ward compatibility then if someone is using the oldest way to upgrade an existing installation the chart will take it into account.

Reply:
@dani8art Can you have a look now? Added a new test if `repositories` not provided then `repository` and `branch` must be provided.

Didn't add `repository` and `branch` to `README.md` nor `values.yaml`.

Bumped minor version to 6.6.0

Reply:
@dani8art Can you have a look now? Added a new test if `repositories` not provided then `repository` and `branch` must be provided.

Didn't add `repository` and `branch` to `README.md` nor `values.yaml`.

Bumped minor version to 6.6.0

Reply:
I would keep all theses tests